src/main/java/com/minecolonies/coremod/entity/citizen/EntityCitizen.java

```diff
@@ -1042,12 +1042,6 @@ private boolean determineDesiredActivity()
         {
             if (desiredActivity == DesiredActivity.SLEEP)
             {
-                if (!getCitizenColonyHandler().getColony().getRaiderManager().isRaided()
-                      && !citizenData.getCitizenMournHandler().isMourning()
-                      && !CompatibilityUtils.getWorldFromCitizen(this).isRaining())
-                {
-            	    citizenData.triggerInteraction(new StandardInteraction(new TranslationTextComponent(COM_MINECOLONIES_COREMOD_ENTITY_CITIZEN_SLEEPING), ChatPriority.HIDDEN));
```

Comment:
Is this still triggered at the appropriate places?

Reply:
ye its slightly different now, triggers when actually sleeping but that fits the message better